pipelines/build/common/openjdk_build_pipeline.groovy

```diff
@@ -315,6 +316,12 @@ class Build {
         List dynamicList = buildConfig.DYNAMIC_LIST
         List numMachines = buildConfig.NUM_MACHINES
         def enableTestDynamicParallel = Boolean.valueOf(buildConfig.ENABLE_TESTDYNAMICPARALLEL)
+        def aqa_branch = "master"
+        def useTestEnv = false
```

Comment:
So this is saying that if the user passes in the SCM ref it's automatically also the aqaBranch and we say we're using a testEnv?

I'm not 100% sure on the logic...